tests/test_benchmark/test_benchmark.py

```diff
@@ -99,8 +109,10 @@ def encode(self, sentences, **kwargs):
 
     if isinstance(task_name, mteb.AbsTask):
         tasks = [task_name]
+        _task_type = task_name.metadata.type
     else:
         tasks = mteb.get_tasks(tasks=[task_name])
+        _task_type = tasks[0].metadata.type
```

Comment:
Not sure what is happening here?

Reply:
I was tasting prompts there in the beginning. Will remove it